h2o-genmodel/src/main/java/hex/genmodel/ModelMojoFactory.java

```diff
@@ -2,17 +2,18 @@
 
 import hex.genmodel.algos.deeplearning.DeeplearningMojoReader;
 import hex.genmodel.algos.drf.DrfMojoReader;
+import hex.genmodel.algos.ensemble.StackedEnsembleMojoReader;
```

Comment:
Are the  `hex.genmodel.algos.ensemble.StackedEnsembleMojoReader` and `hex.genmodel.algos.pca.PCAMojoReader` imports relevant to this PR?

Reply:
I am not sure what is going on, IntelliJ did auto import on my behave.  Super weird.

Reply:
So please, could you remove these imports if there are not related to your changes? Sometimes IntelliJ adds automatic import when I select wrongly autocompleted class, so that is maybe why these imports appeared. :)

Reply:
@wendycwong, WDYT about the removing these imports?